### DISCUSSION TITLE
What is meant by the term 'inflecttive' ?

### DISCUSSION THREAD

**Discussion by metaperl** [Chaoyingz/flect#6] [Q&A] [answered]
In https://flect.celerforge.com/learn/routing/ we read `This makes managing and understanding your application's routes straightforward and inflecttive.` ... but what does  'inflecttive' mean? My guess is the author mean "intuitive".

**Comment by Chaoyingz** [comment, marked as answer]
Thanks for your feedback! You’re right, that was a typo. It should be ‘intuitive.’
